react/features/participants-pane/components/MeetingParticipantItem.js

```diff
@@ -38,6 +39,7 @@ export const MeetingParticipantItem = ({
     onLeave,
     participant
 }: Props) => {
+    const { t } = useTranslation();
```

Comment:
I think there is the need for some params here. Maybe create a custom hook.
Check /jitsi-meet/react/features/base/i18n/functions.js.